examples/reference/pandas/bar.ipynb

```diff
@@ -18,9 +18,7 @@
     "\n",
     "A `bar` plot represents **categorical data** with rectangular bars with heights proportional to the **numerical values** that they represent.\n",
     "The x-axis represents the categories and the y axis represents the numerical value scale.\n",
-    "The bars are of equal width which allows for instant comparison of data.\n",
-    "\n",
-    "Write `hvplot.extension('bokeh')`, `hvplot.extension('matplotlib')` or `hvplot.extension('plotly')` to change the plotting backend"
+    "The bars are of equal width which allows for instant comparison of data."
```

Comment:
Removed the mention to the other extensions. If we do it, it's something that should happen all at once to make sure we do it consistently.